src/styles/components/buttons/variables.less

```diff
@@ -0,0 +1,11 @@
+/* -----------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+banner-plugin/variables.less
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+----------------------------------------------------------------------------- */
+
+
+
+@buttons-enabled:                                                               true;
```

Comment:
I would suggest overriding variables for each component, because it's possible that the author of CNVS may change the variable name, or decide to ship with buttons disabled.